modules/lang-painless/src/main/java/org/elasticsearch/painless/node/SFunction.java

```diff
@@ -108,10 +86,16 @@ void storeSettings(CompilerSettings settings) {
         this.settings = settings;
     }
 
+    void extractVariables() {
```

Comment:
couldn't the few callers of this just call this method themselves with an empty set?

Reply:
Yes, that makes more sense.  Will fix this.